src/Components/Calculator.js

```diff
@@ -1,39 +1,44 @@
-import './Calculator.css';
-import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import './calculator.css';
```

Comment:
- [ ] [REQUIRED] Good job so far! There is an `error` while running the project, please fix it 🙏🏼  Kindly replace the `import './calculator.css';`❌ to `import './Calculator.css';` ✔️  

![image](https://user-images.githubusercontent.com/63915024/234627494-b5e22cf9-789d-4a67-8ddd-3d32391d7f67.png)